examples/powertools-examples-core/README.md

```diff
@@ -1,25 +1,32 @@
-#  Powertools for AWS Lambda (Java) - Core Utilities Example
+#  Powertools for AWS Lambda - Core Utilities Example
```

Comment:
I would keep the Java even if it's kotlin

Reply:
Makes sense, since the library is still native-Java